src/sys/resource.rs

```diff
@@ -0,0 +1,69 @@
+use std::mem;
+
+use libc::{self, c_int};
+pub use libc::{rlimit, rlim_t, RLIM_INFINITY};
```

Comment:
I understand why you're reexporting `rlim_t`, but why reexport the others?

Reply:
Done, limited the reexport to `rlim_t`